src/apps/schedule/service/schedule.py

```diff
@@ -291,19 +291,34 @@ async def get_public_all_schedules(
             flow_id = await FlowEventsCRUD(self.session).get_by_event_id(
                 event_id=event.id
             )
-            notification = await self._get_notifications_and_reminder(event.id)
+            base_event = BaseEvent(**event.dict())
 
-            events.append(
-                PublicEvent(
-                    **event.dict(),
-                    periodicity=PublicPeriodicity(**periodicity.dict()),
-                    respondent_id=None,
+            full_events.append(
+                EventFull(
+                    id=event.id,
+                    **base_event.dict(),
+                    periodicity=periodicity,
                     activity_id=activity_id,
                     flow_id=flow_id,
-                    notification=notification,
                 )
             )
 
+        events = PublicEventByUser(
+            applet_id=applet_id,
```

Comment:
Aren't you exposing the applet_id this way? I guess you could use `list[ScheduleEventDto]` instead of `PublicEventByUser`. So that you don't end up with applet_id, and have similar output as `users/me/events/{applet_id}`

Reply:
Yes, I do expose. But this is not an issue btw :)
Applet ID is not a sensitive data in our case

Reply:
Well, that's just suggestion. You can resolve conversation if this approach if fine by everyone.